pdr_backend/pdr_dashboard/pages/feeds.py

```diff
@@ -216,9 +216,20 @@ def get_feeds_subscription_stat_with_contract(
         result = find_with_key_value(feed_subcription_stats, "contract", contract)
 
         if result:
+            df_buy_count_str = (
+                f"({result['df_buy_count']}-DF)" if result["df_buy_count"] > 0 else ""
+            )
+            ws_buy_count_str = (
+                f"({result['ws_buy_count']}-WS)" if result["ws_buy_count"] > 0 else ""
+            )
+
+            sales_str = (
+                f"{result['sales']} {df_buy_count_str} {ws_buy_count_str}".strip()
+            )
+
```

Comment:
Instead of displaying it as `266 (264-DF) (2-WS)` display it as `266 (264-DF, 2-WS)`